src/flows/context/flow.list.tsx

```diff
@@ -209,6 +215,8 @@ export const FlowListProvider: FC = ({children}) => {
         data: flow.data,
         meta: flow.meta,
         readOnly: flow.readOnly,
+        createdAt: flow.createdAt ?? new Date().toISOString(),
+        updatedAt: flow.updatedAt ?? new Date().toISOString(),
```

Comment:
same as above, figured this could be a fail-safe in case the metadata doesn't exist?